site/content/docs/5.3/components/tooltips.md

```diff
@@ -130,7 +130,7 @@ With an SVG:
 <!--Boosted mod: added an aria-label="Default tooltip" because the title is automatically removed by JS-->
 
 <div class="bd-example tooltip-demo">
-  <a href="#" class="d-inline-block" data-bs-toggle="tooltip" data-bs-title="Default tooltip" aria-label="Default tooltip">
+  <a href="#" class="d-inline-block" data-bs-toggle="tooltip" data-bs-title="Default tooltip" aria-label="Focus or hover to open default tooltip">
```

Comment:
Maybe we can remove this modification from this PR as it's not really the same modification. And that will come back to Boosted automatically as soon as https://github.com/twbs/bootstrap/pull/40756/files#r1750127676 is taken into account?

Reply:
I moved the change to PR https://github.com/twbs/bootstrap/pull/40808